File: C++/Fibonacci/fibonacci.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
	int n = 10;
	int num[n];
	
	if (n > 1)
	{
		num[0] = 1;
	}
	if (n > 2)
	{
		num[1] = 1;
	}
	for (int i = 2; i < n; i++)
	{
		num[i] = num[i-1] + num[i-2];
		
	}	

	for (int k = 0; k < n; k++)
	{
		cout << to_string(k+1) + " : " + to_string(num[k]) + "\n";
	}
	
	return 0;

}

File: Julia/Fibonacci/Fibonacci.jl
function fibonacci(n)

	num = fill(0.0, n)

	if n > 0
		num[1] = 1
	end

	if n > 1
		num[2] = 1
	end
	
	if n > 2
		for k = 3:n
			num[k] = num[k - 1] + num[k - 2]
		end
	end
	
	return num

end

n = 10
x = fibonacci(n)

println(x)

